<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpperComputer</class>
 <widget class="QMainWindow" name="UpperComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能停车场</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_everyPark">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>771</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>各车位状态</string>
    </property>
    <widget class="QGroupBox" name="groupBox_park1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>200</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Park 1</string>
     </property>
     <widget class="QLabel" name="label_park1_statu_pic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>140</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_park1_statu">
           <property name="text">
            <string>状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park1_statu_text">
           <property name="text">
            <string>Occupied</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_park1_carNum">
           <property name="text">
            <string>车牌号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park1_carNum_text">
           <property name="text">
            <string>EA12345</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_park1_inTime">
           <property name="text">
            <string>入位时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park1_inTime_text">
           <property name="text">
            <string>2017-7-21 09:59:50</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_park1_call">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_park2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>200</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Park 2</string>
     </property>
     <widget class="QLabel" name="label_park2_statu_pic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>140</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_park2_statu">
           <property name="text">
            <string>状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park2_statu_text">
           <property name="text">
            <string>Occupied</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_park2_carNum">
           <property name="text">
            <string>车牌号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park2_carNum_text">
           <property name="text">
            <string>EA12345</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_park2_inTime">
           <property name="text">
            <string>入位时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park2_inTime_text">
           <property name="text">
            <string>2017-7-21 10:04:46</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_park2_call">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_park3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>200</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Park 3</string>
     </property>
     <widget class="QLabel" name="label_park3_statu_pic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>140</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_park3_statu">
           <property name="text">
            <string>状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park3_statu_text">
           <property name="text">
            <string>Free</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_park3_carNum">
           <property name="text">
            <string>车牌号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park3_carNum_text">
           <property name="text">
            <string>NONE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_park3_inTime">
           <property name="text">
            <string>入位时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park3_inTime_text">
           <property name="text">
            <string>NONE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_park3_call">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_park4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>200</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Park 4</string>
     </property>
     <widget class="QLabel" name="label_park4_statu_pic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>140</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_park4_statu">
           <property name="text">
            <string>状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park4_statu_text">
           <property name="text">
            <string>Waitting</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_park4_carNum">
           <property name="text">
            <string>车牌号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park4_carNum_text">
           <property name="text">
            <string>EA12345</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_park4_inTime">
           <property name="text">
            <string>入位时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park4_inTime_text">
           <property name="text">
            <string>NONE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_park4_call">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_park5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>370</y>
       <width>200</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Park 5</string>
     </property>
     <widget class="QLabel" name="label_park5_statu_pic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>140</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_park5_statu">
           <property name="text">
            <string>状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park5_statu_text">
           <property name="text">
            <string>Occupied</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_park5_carNum">
           <property name="text">
            <string>车牌号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park5_carNum_text">
           <property name="text">
            <string>EA12345</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_park5_inTime">
           <property name="text">
            <string>入位时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park5_inTime_text">
           <property name="text">
            <string>2017-7-21 10:04:46</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_park5_call">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_park6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>370</y>
       <width>200</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Park 6</string>
     </property>
     <widget class="QLabel" name="label_park6_statu_pic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>140</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_park6_statu">
           <property name="text">
            <string>状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park6_statu_text">
           <property name="text">
            <string>Waitting</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_park6_carNum">
           <property name="text">
            <string>车牌号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park6_carNum_text">
           <property name="text">
            <string>EA12345</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_park6_inTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>入位时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_park6_inTime_text">
           <property name="text">
            <string>NONE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_park6_call">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>320</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>360</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>实时信息</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_Information">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Hum">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>停车场湿度</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Temp">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>360</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>停车场温度</string>
    </property>
    <widget class="QLabel" name="label_Temperature">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>161</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Alarm">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>50</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>停车场状态</string>
    </property>
    <widget class="QLabel" name="label_Warning">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>管理员操作</string>
    </property>
    <addaction name="action_signUp"/>
    <addaction name="action_recharge"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_signUp">
   <property name="text">
    <string>新用户注册</string>
   </property>
  </action>
  <action name="action_recharge">
   <property name="text">
    <string>充值</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
